java/mlt-cli/src/main/java/org/maplibre/mlt/cli/Encode.java

```diff
@@ -488,6 +495,13 @@ private static void encodeMBTiles(
         try {
           while (tiles.hasNext()) {
             final var tile = tiles.next();
+            if (tile.getZoom() > maxzoom) {
```

Comment:
Are tiles ordered by zoom level so this won't miss tiles in lower zoom level accidentally?

Reply:
The question around this comment is if the fact that the order is different cause the code to stop before processing all the tile?